src/bugreporter/odemis_bugreporter.py

```diff
@@ -634,13 +638,15 @@ def _on_report_sent_successful(self):
 
     def open_failed_upload_dlg(self):
         """
-        Ask the user to user wetransfer in case the upload to osticket failed.
+        Ask the user to use the website in case the upload to osticket failed.
         """
-        txt = ('The bug-report could not be uploaded to osticket. Please finish the report ' +
-               'by filling in the form on https://support.delmic.com and attaching the report ' +
-               'file "%s" on the Desktop.\n\n' % self.bugreporter.zip_fn +
-               'After closing this window, the form will automatically open in your web browser.')
-        dlg = wx.MessageDialog(self, txt, '', wx.OK)
+        txt = ('The bug-report could not be uploaded to osticket. Please finish the report '
+               'by filling in the form on https://support.delmic.com and attaching the ZIP '
+               'file "%s" on the Desktop.\n\n'
+               'Alternatively, you can send the file to support@delmic.com .\n\n'
+               'After closing this window, the form will automatically open in your web browser.' %
```

Comment:
I would use the passive form here: `...will be automatically opened in your ...`